content/doc/developer/building/index.adoc

```diff
@@ -8,6 +8,8 @@ references:
   title: Why Maven?
 - url: https://wiki.jenkins-ci.org/display/JENKINS/Debugging+native+Maven+jobs
   title: Debugging native Maven jobs # TODO convert to guide
+- url: https://www.jimklimov.com/2019/02/debugging-jenkins-plugins-and-core-with.html
+  title: One contributor's experience tracing Jenkins plugins and the Jenkins server itself under the Netbeans IDE # TODO convert to guide
```

Comment:
Does the TODO get ignored correctly ?

Reply:
```suggestion
  title: Tracing Jenkins plugins and the Jenkins server itself under the Netbeans IDE # TODO convert to guide
```

Reply:
> Does the TODO get ignored correctly ?

It seems to be ignored well for the previous item, so I copied it :)

> Suggested change

Might do, though the blog entry has more of a "how it worked for me" rather than a blueprint or definitive guide doc :-\ Maybe adding a "(blog)" disclaimer into the title along with your shortening would do better then.